ISISScientificComputing/autoreduce
# First system test

Adds a new system test only for GEM
We will add new instruments in future PRs as part of #138 

To test:
Check the following works correctly locally
```
sudo python setup.py start
pytest -m systemtest
```
This should also skip the test on Windows
